docs/MultiViewOpenEXR.rst

```diff
@@ -44,17 +42,138 @@ Stereo viewing software may default to assuming that it should send the
 ``left`` and ``right`` views to the left and right eyes respectively, and
 may ignore all other views.
 
-Other view names should be kept as short as possible to keep the channel
-names within OpenEXR's 31-character limit.
+Other view names should be kept as short as possible
+
+
+Single Part and Multi-Part Conventions
+======================================
+
+This document describes two schemes. The preferred scheme requires
+multi-part file support, which was introduced in OpenEXR-2.0.
+An older scheme uses a channel naming scheme to store multiple views
+in a single part file.
+
+In some cases, the single-part scheme may give better performance in
+stereo playback systems than the multi-part scheme, so may still be advantageous.
+Since such files are increasingly rare, it is strongly recommended that the multi-part scheme
+is used when sharing files with others and for long-term storage.
+
+
+Storing Multi-View Images in Multi-Part OpenEXR Files
+=====================================================
+
+Multi-part EXR files use the ``view`` attribute within the header for each part
+to indicate that all the channels within that part are associated with that view.
+A stereo EXR will contain part(s) with the ``view`` attribute set to ``left`` and other part(s)
+with the ``view`` attribute set to ``right``
+
+*The conventions here should not be combined*: if a file contains more than one part,
```

Comment:
Suggestion, after re-reading this a few times:

*The multi-part and multi-view conventions described here should not be combined*